libs/payments/ui/src/lib/client/components/PaymentMethodManagement/en.ftl

```diff
@@ -0,0 +1,7 @@
+## Payment method management page - Stripe
+# Save button for changing which payment method will be used
+payment-method-management-save-default = Use for future payments
+# Save button for saving a new payment method
+payment-method-management-save-method = Save payment info
+
+manage-stripe-payments-title = Manage payment methods
```

Comment:
```suggestion
manage-stripe-payments-title = Manage payment methods

##
```

Reply:
Thanks -- looks like the `##` tag is an open/close section operator?